pmgr/pmgrAPI.py

```diff
@@ -339,3 +339,56 @@ def add_hutch(self, newhutch):
         el = self.pm.end_transaction()
         if el != []:
             raise Exception("DB Errors", el)
+
+    @staticmethod
+    def _get_datetime(s):
+        """
+        Try to convert a string into a datetime object.  The following
+        conversions are attempted:
+             ISO format
+             mm/dd/yyyy
+             mm/dd/yyyy hh:mm
+        If all three fail, a ValueError exception is raised.
```

Comment:
This docstring line isn't correct based on the as-implemented code

Reply:
Agreed.